relay-server/src/utils/sizes.rs

```diff
@@ -65,6 +65,7 @@ pub fn check_envelope_size_limits(config: &Config, envelope: &Envelope) -> bool
             ItemType::UserReport => (),
             ItemType::Statsd => (),
             ItemType::MetricBuckets => (),
+            ItemType::MetricMeta => (),
```

Comment:
For follow-up: Let's add a size limit for metrics and meta. The limit is declared in the main `Config`.